src/js/components/put-away/SplitLineModal.jsx

```diff
@@ -132,10 +151,18 @@ class SplitLineModal extends Component {
    * higher than available one.
    * @public
    */
-  isValid() {
+
+  isNegativeQuantity() {
+    return _.some(this.state.splitItems, items => _.toInteger(items.quantity) <= 0);
+  }
+
+  isQuantityHigherThanOriginalPutaway() {
     const qtySum = this.calculatePutAwayQty();
+    return qtySum > _.toInteger(this.props.putawayItem.quantity);
+  }
 
-    return qtySum <= _.toInteger(this.props.putawayItem.quantity);
+  isValid() {
+    return !this.isNegativeQuantity() && !this.isQuantityHigherThanOriginalPutaway();
```

Comment:
do you use it anywhere? I see that in line `228` you delete `disabled={this.isValid()}` where it was used and I can't see any usage of this anymore?

Reply:
I used it in line 328 `disabled={!this.isValid() || !this.isBinSelected()}`